ExecuteJobs.java

```diff
@@ -0,0 +1,80 @@
+import java.util.*;
+import java.io.*;
+
+public class ExecuteJobs {
+  public long start(String process) throws IOException {
+    // TODO some error handling
+    Process p = Runtime.getRuntime().exec(process);
+    long pid = p.pid();
+    return pid;
+  }
+
+  public int stop(long pid) throws IOException {
+    Process p = Runtime.getRuntime().exec("ps -q " + pid + " -o state --no-headers");
+    BufferedReader pInput = new BufferedReader(new InputStreamReader(p.getInputStream()));
+    String s = pInput.readLine();
+    if(s == null) {
+      // pid not found
+      return 2;
+    }
+
+    Runtime.getRuntime().exec("kill " + pid);
+    p = Runtime.getRuntime().exec("ps -q " + pid + " -o state --no-headers");
+    pInput = new BufferedReader(new InputStreamReader(p.getInputStream()));
+    s = pInput.readLine();
+    if(s != null) {
+      for(int i = 0; i < 5; i++) {
```

Comment:
why 5?

Reply:
5 is arbitrary. I want to have a few chances if kill pid doesn't work on the first try. I think when I originally implemented this, I also intended to put a time delay in at the end of each for loop iteration.

Reply:
I wold rather fail and let the client do the retries for the purposes of this task.

Reply:
Okay. I'll remove this piece. This will also eliminate the issue with indentation levels that you mentioned in a different comment.